go-controller/pkg/node/node.go

```diff
@@ -845,7 +845,12 @@ func upgradeServiceRoute(bridgeName string) error {
 }
 
 // DummyNextHopIPs returns the fake next hops used for service traffic routing when
-// a node has no default gateway
+// a node has no default gateway.
```

Comment:
Hmm, when I think about it this kind of comment would make it easier to understand for me 
```
// DummyNextHopIPs returns the fake next hops used for service traffic routing, used in:
// - br-ex, where we don't really care about the next hop GW in use as traffic is always routed to OVN
// - OVN, only when there is no default GW as it wouldn't matter since there is no external traffic
```
WDYT?

Reply:
Thanks for the providing a more accurate and less wordy explanation.

Reply:
Done.